.github/CODEOWNERS

```
@@ -40,7 +40,7 @@ exporter/azuredataexplorerexporter/                  @open-telemetry/collector-c
 exporter/carbonexporter/                             @open-telemetry/collector-contrib-approvers @pjanotti
 exporter/clickhouseexporter/                         @open-telemetry/collector-contrib-approvers @hanjm @dmitryax
 exporter/coralogixexporter/                          @open-telemetry/collector-contrib-approvers @oded-dd @ofirshmuel
-exporter/datadogexporter/                            @open-telemetry/collector-contrib-approvers @KSerrania @mx-psi @gbbr @knusbaum @amenasria @dineshg13
+exporter/datadogexporter/                            @open-telemetry/collector-contrib-approvers @KSerrania @mx-psi @gbbr @knusbaum @amenasria @dineshg13 @liustanley
```

We can also take this opportunity to remove @KSerrania @knusbaum and @amenasria  from here since they are no longer actively working on the exporter (will reach out for their approvals)

```suggestion
exporter/datadogexporter/                            @open-telemetry/collector-contrib-approvers @mx-psi @gbbr @dineshg13 @liustanley
```

---

works for me, yes.

---

Hey, yes you can remove me, no prob !! 🫡

---

Looks good to me.